setup.py

```diff
@@ -182,7 +182,7 @@ def read(*names, **kwargs):
 CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
 
 HELICS_SOURCE = os.path.join(CURRENT_DIRECTORY, "./_source")
-PYHELICS_INSTALL = os.path.join(CURRENT_DIRECTORY, "./helics/install")
+PYHELICS_INSTALL = os.environ.get("PYHELICS_INSTALL") if os.environ.get("PYHELICS_INSTALL") else os.path.join(CURRENT_DIRECTORY, "./helics/install")
```

Comment:
You can do the same thing like this instead:

```suggestion
PYHELICS_INSTALL = os.environ.get("PYHELICS_INSTALL", os.path.join(CURRENT_DIRECTORY, "./helics/install")
```

```python
$ ipython
In [1]: import os

In [2]: os.environ.get("PYHELICS_INSTALL", "testing")
Out[2]: 'testing'
```

```python
$ export PYHELICS_INSTALL="./this/folder"

$ ipython
In [1]: import os

In [2]: os.environ.get("PYHELICS_INSTALL", "testing")
Out[2]: './this/folder'
```